cc.retroshare.retroshare-gui.appdata.xml

```diff
@@ -32,6 +32,7 @@
   <url type="homepage">http://retroshare.cc/</url>
   <updatecontact>contact_at_retroshare.cc</updatecontact>
   <releases>
+    <release data="2021-03-23" version="0.6.6"/>
```

Comment:
```suggestion
    <release date="2021-03-23" version="0.6.6"/>
```

Reply:
Damn, I was hoping I'd be quick enough to push my fix before someone noticed :P

Reply:
I make that typo pretty often.